pkg/database/sorted_set.go

```diff
@@ -258,32 +256,33 @@ func (d *db) VerifiableZAdd(req *schema.VerifiableZAddRequest) (*schema.Verifiab
 		return nil, store.ErrIllegalArguments
 	}
 
-	txMetatadata, err := d.ZAdd(req.ZAddRequest)
+	lastTx, err := d.allocTx()
```

Comment:
it may be moved some lines below, right before its use

Reply:
smae comment as in case of references - this is to avoid hard-to-interpret errors on high load